ui/pages/asset/components/token-buttons.tsx

```diff
@@ -0,0 +1,313 @@
+import React, { useContext, useEffect } from 'react';
```

Comment:
This file just extracted the buttons from the old `token-asset.js`.  No changes other than converted to typescript.